solutions/instances/variables.tf

```diff
@@ -168,126 +168,126 @@ variable "enable_platform_metrics" {
   type        = string
   default     = "observability-cos"
-  description = "The name to use when creating the Cloud Object Storage instance. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the Cloud Object Storage instance to create. If the prefix input variable is passed, the name of the instance is prefixed to the value in the `<prefix>-value` format."
 }
 
 variable "cos_instance_tags" {
   type        = list(string)
-  description = "Optional list of tags to be added to Cloud Object Storage instance. Only used if not supplying an existing instance."
+  description = "A list of optional tags to add to a new Cloud Object Storage instance."
   default     = []
 }
 
 variable "cos_instance_access_tags" {
   type        = list(string)
-  description = "A list of access tags to apply to the Cloud Object Storage instance. Only used if not supplying an existing instance."
+  description = "A list of access tags to apply to a new Cloud Object Storage instance."
   default     = []
 }
 
 variable "log_archive_cos_bucket_name" {
   type        = string
   default     = "log-archive-cos-bucket"
-  description = "The name to use when creating the Cloud Object Storage bucket for storing log archives (NOTE: bucket names are globally unique). If 'add_bucket_name_suffix' is set to true, a random 4 characters will be added to this name to help ensure bucket name is globally unique. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the Cloud Object Storage bucket to create to store log archive files. Cloud Object Storage bucket names are globally unique. If the `add_bucket_name_suffix` variable is set to `true`, 4 random characters are added to this name to ensure that the name of the bucket is globally unique. If the prefix input variable is passed, the name of the instance is prefixed to the value in the `<prefix>-value` format."
 }
 
 variable "at_cos_target_bucket_name" {
   type        = string
   default     = "at-events-cos-bucket"
-  description = "The name to use when creating the Cloud Object Storage bucket for cos target (NOTE: bucket names are globally unique). If 'add_bucket_name_suffix' is set to true, a random 4 characters will be added to this name to help ensure bucket name is globally unique. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the Cloud Object Storage bucket to create for the Cloud Object Storage target. Cloud Object Storage bucket names are globally unique. If the `add_bucket_name_suffix` variable is set to `true`, 4 random characters are added to this name to ensure that the name of the bucket is globally unique. If the prefix input variable is passed, the name of the instance is prefixed to the value in the `<prefix>-value` format."
```

Comment:
Please add AT events too. It was missed before as well.
First line in description can be - 
`The name of the Cloud Object Storage bucket to create for the Cloud Object Storage target to store AT events.`